docs/terms/expectation.md

```diff
@@ -3,10 +3,134 @@ title: Expectation
+
+
+## Features
+
+### Versatility out of the box
+
+Great Expectations' built-in library includes more than 50 common Expectations, such as:
+
+* `expect_column_values_to_not_be_null`
+* `expect_column_values_to_match_regex`
+* `expect_column_values_to_be_unique`
+* `expect_column_values_to_match_strftime_format`
+* `expect_table_row_count_to_be_between`
+* `expect_column_median_to_be_between`
+
+For a full list of available Expectations, please check out the [Expectation Gallery](https://greatexpectations.io/expectations).
+
+### Customization
+
+Expectations are especially useful when they capture critical aspects of data understanding that analysts and practitioners know based on its *semantic* meaning. It's common to want to extend Great Expectations with application or domain specific Expectations. For example:
+
+```bash
+expect_column_text_to_be_in_english
+expect_column_value_to_be_valid_icd_code
+```
+
+These Expectations aren't included in the default set, but could be very useful for specific applications.
+
+Fear not! Great Expectations is designed for customization and extensibility.
+
+Building custom Expectations is easy and allows your custom logic to become part of the validation, documentation, and even profiling workflows that make Great Expectations stand out. See the guide on [creating custom Expectations](/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_expectations) for more information on building Expectations and updating <TechnicalTag relative="../" tag="data_context" text="Data Context" /> configurations to automatically load <TechnicalTag relative="../" tag="batch" text="Batches" /> of data with custom <TechnicalTag relative="../" tag="data_asset" text="Data Assets" />Data Assets.
+
+### Distributional Expectations
+
+Distributional Expectations rely on expected distributions or "partition objects", which are built from intervals for continuous data or categorical classes and their associated weights, in order to help identify when new datasets or samples may be different than expected.  You should use Distributional Expectations in the same way as other Expectations: to help accelerate identification of risks and changes to a modeled system or disruptions to a complex upstream data feed.
+
+For more information, please see our [reference guide on Distributional Expectations](../reference/expectations/distributional_expectations.md).
+
+### Conditional Expectations
```

Comment:
I'm not sure it's worth diving into the detail, but to be explicit: all map (row-wise) expectations can be conditional. Currently this conditionality is only implemented for pandas. So whereas Distributional Expectations represent specific expectation types (e.g. `expect_column_kl_divergence_to_be_less_than`), Conditional Expectations represent a facet of map expectations (including things like `expect_column_values_to_be_in_set`).

Again, this detail might not be important here, but wanted to drop this for reference.

Reply:
I've added your expanded explanations to the Conditional and Distributional Expectations sections.  These changes will appear in the next commit.  The fact that Conditional Expectations are only available in Pandas is specified under the Limitations section, right below the Conditional Expectations section, but I've also added a note about it inside of the Conditional Expectations section.